app/scripts/utils/localStorage.js

```diff
@@ -30,11 +33,13 @@ function doOperation (op) {
     if (op === 'getItem') {
       try {
         result = JSON.parse(result)
-      } catch (e) {}
+      } catch (e) {
+        log('Unable to parse result as json. Message: %s', e.message)
+      }
     }
 
     return result
   } catch (e) {
-    console.warn('LocalStorage error: ' + e.message)
+    console.warn(e.message)
```

Comment:
But with a warn level: `const warn = debug('utils:localStorage:warn')` or sth like that